website/templates/index/efficient.html

```diff
@@ -4,11 +4,12 @@
             <div class="col-lg">
                 <h2 id="hardware-efficient" class="my-4">Hardware efficient</h2>
                 <p class="lead">ClickHouse processes typical analytical queries two to three orders of magnitude faster than traditional
-            row-oriented systems with the same available I/O throughput and CPU capacity. The system's columnar storage format allows fitting
+            row-oriented systems with the same available I/O throughput and CPU capacity. Columnar storage format allows fitting
             more hot data in RAM, which leads to a shorter typical response times.</p>
+                <p class="lead">Total cost of ownership could be further lowered by using commodity hardware with rotating disk drives instead of enterprise grade NVMe or SSD without significant sacrafices in latency for most kinds of queries.</p>
```

Comment:
```suggestion
                <p class="lead">Total cost of ownership could be further lowered by using commodity hardware with rotating disk drives instead of enterprise grade NVMe or SSD without significant sacrifices in latency for most kinds of queries.</p>
```